README.md

```diff
@@ -96,7 +96,7 @@ vars:
 Fivetran offers the ability for you to orchestrate your dbt project through [Fivetran Transformations for dbt Core™](https://fivetran.com/docs/transformations/dbt). Learn how to set up your project for orchestration through Fivetran in our [Transformations for dbt Core setup guides](https://fivetran.com/docs/transformations/dbt#setupguide).
 </details>
 
-# 🔍 Does this package have dependencies?
+# 🔍 Does this package have dependencies? 
```

Comment:
Reminder to add spark-utils if it is needed after making the package databricks compatible.

Reply:
added

Reply:
It actually looks like you didn't include spark-utils in the packages.yml. If spark-utils is not needed then we won't need to include it in the dep list.

That being said, you likely will need it in the transform package so it may be worthwhile to also include here. Regardless, be sure to add spark-utils to the packages.yml.

Reply:
thanks for catching this, added to packages.yml